.gitignore

```diff
@@ -157,7 +157,9 @@ packages/fxa-settings/test/
 
 # payments-next
 apps/payments/next/public/locales
-apps/payments/next/lib/metrics/glean/server_events.ts
+
+# payments-metrics
+libs/payments/metrics/src/lib/glean/__generated__/server_events.ts
```

Comment:
`apps/payments/next/lib/metrics/glean/server_events.ts` auto-generated when I pulled down the branch.

Reply:
`apps/payments/next/lib/metrics/glean/server_events.ts` was the original location of `server_events.ts` so it might just be an artifact from before this PR.

If you delete the folder `apps/payments/next/lib/metrics` it should not be automatically created again.